packages/react-server-cli/.nsprc

```diff
@@ -1,3 +1,3 @@
 {
-  "exceptions": ["https://nodesecurity.io/advisories/99"]
+  "exceptions": ["https://nodesecurity.io/advisories/106"]
```

Comment:
I have to add the exemption due to https://github.com/expressjs/serve-index/issues/56